core/modules/qproc/SecondaryIndex.cc

```diff
@@ -129,39 +129,49 @@ class MySqlBackend : public SecondaryIndex::Backend {
         std::vector<std::string> const& params,
         QueryType const& query_type) {
 
-
         LOGS(_log, LOG_LVL_TRACE, "params: " << util::printable(params));
 
-        std::string const& db = params[0];
-        std::string const& table = params[1];
-        std::string const& key_column = params[2];
+        auto iter = params.begin();
+        std::string const& db = *iter; // params[0]
+        ++iter;
```

Comment:
This is not strictly require. But these two lines could be combined into just:

std::string const& db= *(iter++);  // param[0]